presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,213 @@
+===================
+LDAP Authentication
+===================
+
+Presto can be configured to enable frontend LDAP authentication over
+HTTPS for clients, such as the :ref:`cli_ldap`, or the JDBC and ODBC
+drivers. This involves user and password validation backed by LDAP.
+
+To enable LDAP authentication for Presto, configuration changes are made on
+the Presto coordinator. No changes are required to the worker configuration;
+only the communication from the clients to the coordinator is authenticated.
+
+Presto Server Configuration
+---------------------------
+
+Environment Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _ldap_server:
+
+LDAP Server
+~~~~~~~~~~~
+
+The Presto clients send a user name and password to the coordinator and
+coordinator validates these credentials using an external LDAP service.
+The LDAP implementation types supported by Presto are Active Directory
+and OpenLDAP.
```

Comment:
s/LDAP implementation/LDAP server implementation. Or you could rewrite it as:
You can configure Presto coordinator to communicate with one of the following two types of LDAP server implementations:
* Active Directory
* OpenLDAP